stable/nextcloud/README.md

```diff
@@ -121,6 +121,7 @@ The following table lists the configurable parameters of the nextcloud chart and
 | `persistence.annotations`                                    | PVC annotations                                         | `{}`                                                    |
 | `persistence.storageClass`                                   | PVC Storage Class for nextcloud volume                  | `nil` (uses alpha storage class annotation)             |
 | `persistence.existingClaim`                                  | An Existing PVC name for nextcloud volume               | `nil` (uses alpha storage class annotation)             |
+| `persistence.existingClaims`                                 | List of existing PVCs to mount                          | `{}`                                                    |
```

Comment:
Can you please rename this to persistence.extraVolumes? This is common in helm stable charts. 
And I would suggest to use it as an array. With this an integration like the following it is possible to enable all possible settings on kubernetes mounts.

```
{{- if .Values.extraVolumes }}
{{ toYaml .Values.extraVolumes \| indent 6 }}
{{- end }}
```